db/schema.rb

```diff
@@ -193,6 +193,14 @@
     t.index ["path"], name: "index_locations_on_path", unique: true
   end
 
+  create_table "plays", force: :cascade do |t|
```

Comment:
This table will probably be the largest table in a lot of setups. Since we will almost always query this by both user and track, it might be worth it to set an index combining `track_id` and `user_id`.